skyreader/result.py

```diff
@@ -80,7 +80,10 @@ class SkyScanResult:
     """
 
     PIXEL_TYPE = np.dtype(
-        [("index", int), ("llh", float), ("E_in", float), ("E_tot", float)]
+        [
+            ("index", int), ("llh", float), ("E_in", float), ("E_tot", float),
+            ("X", float), ("Y", float), ("Z", float), ("time", float)
```

Comment:
```suggestion
            ("X", float), ("Y", float), ("Z", float), ("T", float)
```

To prevent confusion with time as MJD.